### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by minwoox**
- Dropwizard 2.0.12 -> 2.0.13
- gRPC 1.31.1 -> 1.32.1
- grpc-kotlin-stub 0.1.5 -> 0.2.0
- Micrometer 1.5.4 -> 1.5.5
- Netty 4.1.51.FINAL -> 4.1.52.FINAL
  - netty-tcnative-boringssl-static 2.0.31.Final -> 2.0.34.Final
- Tomcat 9.0.37 -> 9.0.38
- org.jetbrains.kotlinx 1.3.8 -> 1.3.9
- Spring 5.2.8.RELEASE -> 5.2.9.RELEASE
- Spring Boot 2.3.3.RELEASE -> 2.3.4.RELEASE
- Build
  - akka-actor-typed_2.13 2.6.8 -> 2.6.9
  - Checkstyle 8.35 -> 8.36.1
  - Eureka 1.9.25 -> 1.9.26
  - Finagle 20.8.0 -> 20.8.1
  - jmh-core 1.25 -> 1.25.2
  - jmh-gradle-plugin 0.5.0 -> 0.5.1
  - json-unit 2.18.1 -> 2.19.0
  - Junit 5.6.2 -> 5.7.0
  - ktlint-gradle 9.3.0 -> 9.4.0
  - Mockito 3.5.2 -> 3.5.11
  - testNG 7.1.0 -> 7.3.0
- Examples
  - Dagger 2.28.3 -> 2.29.1
  - monix-reactive_2.13 3.2.2+43-36c1ada8 -> 3.2.2+45-5c6c8b9e
  - org.jetbrains.kotlin 1.3.72 -> 1.4.10
- Shaded
  - fastutil 8.4.1 -> 8.4.2

Also updated frontend dependencies

### CODE REVIEW

File: dependencies.yml [hunk 1]
```diff
@@ -157,21 +158,21 @@ com.thesamet.scalapb:
...
 
 gradle.plugin.net.davidecavestro:
   gradle-jxr-plugin: { version: '0.2.1' }
 
 io.dropwizard:
   dropwizard-core:
-    version: &DROPWIZARD_VERSION '2.0.12'
+    version: &DROPWIZARD_VERSION '2.0.13'
     javadocs:
     - https://www.javadoc.io/doc/io.dropwizard/dropwizard-core/2.0.12/
```
Comment (ikhoon) [inline dependencies.yml:177]:
```suggestion
    - https://www.javadoc.io/doc/io.dropwizard/dropwizard-core/2.0.13/
```

**Comment by ikhoon** [comment]
> org.jetbrains.kotlin 1.3.72 -> 1.7.40

Maybe org.jetbrains.kotlin 1.3.72 -> 1.4.10 ?

**Comment by minwoox** [comment]
> Maybe org.jetbrains.kotlin 1.3.72 -> 1.4.10 ?

OOps. 😅

**Review by ikhoon — APPROVED** [review]
Thanks! @minwoox 

/cc @okue @gary-lo grpc-kotlin now builds on Windows.

File: dependencies.yml (hunk 1 above)
Comment (minwoox) [reply dependencies.yml]: Fixed. 😉

**Review by trustin — APPROVED** [review]
Thanks, @minwoox! You might want to check the dependency versions again on Monday, though. 😉

**Comment by trustin** [comment]
How about updating the frontend/site dependencies to fix the security vulnerability warnings?

**Comment by minwoox** [comment]
> How about updating the frontend/site dependencies to fix the security vulnerability warnings?

It's already done. 😄

**Pull request merged**